examples/spring-boot-kotlin/build.gradle

```diff
@@ -0,0 +1,46 @@
+buildscript {
+    ext.kotlin_version = '1.3.50'
```

Comment:
How about adding `kotlin-stdlib-jdk8` to `dependencies.yml` and refer to the version via `managedVersion`?